docs/en/source/callFlow.rst

```diff
@@ -102,33 +95,33 @@ Query
+ 2.1. The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
+
+ 2.2. The service API gateway forwards the HTTP request to the Query Manager micro-service.
 
- 2.2. The service API gateway forwards the HTTP request to the Query Manager micro-service.
-   
-3. The query manager now fetches the previously stored data/entities from the Topic “Entities”.
+3. The query manager now fetches the previously stored data/entities from the Topic “ENTITY”.
 
-- If the query is for all entities or specific entities with id and/or attribute is requested, this will be directly served based on Kafka Entity topic data by query manager without involving the storage manager. In short simpler queries like non-geo queries or without regular expression queries associated with entity or entities can be served directly. In this case, the response will be sent back and processing jumps to step 7.2.
+- If the query is for all entities or specific entities with id and/or attribute is requested, this will be directly served based on Kafka Entity topic data by query manager. In short simpler queries like non-geo queries or without regular expression queries associated with entity or entities can be served directly. In this case, the response will be sent back and processing jumps to step 7.2.
 
-- For complex queries, the query manager will take help from the storage manager as mention in the following steps. 
+- For complex queries, the query manager will directly fetch data from DB as mention in the following steps.
 
-4. The Query Manager (in case of complex queries) will publish the query (embedding a used in the message and other metadata) into the Query topic which is being listened by the Storage manager.
+4. Now, our query request firstly gets publish/store in the database and after that further processing is done in the Kafka bus.
 
-5. The storage manager gets the notification for the requested query and starts processing the query over the DB data and builds the query response.
+5. The Query Manager (in case of complex queries) will publish the query (embedding a used in the message and other metadata) into the Query topic.
 
-6. The storage manager publishes the response of query in the Query topic which is being listened by Query manager.
+6. Now, query response is being build after fetching the data from DB for the requested query and afterwards our query response gets published in the query topic which is being listed by Query Manager.
 
-7. The QM receives the notification from the QueryResult topic. 
+7. The QM receives the notification from the QueryResult topic.
 
- 7.1.  It sends the HTTP response back to the API gateway.
+ 7.1. It sends the HTTP response back to the API gateway.
 
- 7.2.API gateway sends back the response to the end-user/requestor.
+ 7.2. API gateway sends back the response to the end-user/requestor.
```

Comment:
[Minor] Add remove all the topic direction like as Query Result, CSource from flow diagram 3 and changes/remove from flow diagram theoretical part as well.

Reply:
Done